packages/composites/bitcoin-json-rpc/schemas/env.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/bitcoin-json-rpc-adapter.json",
+  "title": "@chainlink/bitcoin-json-rpc-adapter env var schema",
+  "required": [],
```

Comment:
```suggestion
  "required": ["RPC_URL"],
```

Reply:
Actually this might get pulled in by "$ref": "https://external-adapters.chainlinklabs.com/schemas/ea-reference-data-reader.json". Resolve if so.

Reply:
Ill include this in the `json-rpc-adapter` parent dep